explorer/explorertypes.go

```diff
@@ -533,3 +533,34 @@ func AddressPrefixes(params *chaincfg.Params) []AddrPrefix {
 func GetAgendaInfo(agendaId string) (*agendadb.AgendaTagged, error) {
 	return agendadb.GetAgendaInfo(agendaId)
 }
+
+// StatsInfo represents the all data for the stats page
+type StatsInfo struct {
+	UltimateSupply             int64
+	TotalSupply                int64
```

Comment:
`TotalSupply` is confusing to me.  This is the *current* supply?  As of what block?  Where is the best block height and hash in this structure to indicate that this is valid for?

Reply:
Totalsupply is CoinSuppy, I could probably put in the block height and the hash but i was following the layout. Do have an idea of where i can put this information. I felt like it might have needed a new card (But that would break the layout) or should i just add it to the totalsupply card.

Reply:
Yes the best block height and hash should definitely be on the stats page.  Maybe not a card.  Perhaps try some text above the "column" headings that gives the chain status.